modules/builds-source-secret-ssh-key-auth.adoc

```diff
@@ -32,5 +32,17 @@ the public key. The private key is used to access your private repository.
 ----
 $ oc create secret generic <secret_name> \
     --from-file=ssh-privatekey=<path/to/ssh/private/key> \
+    --from-file=<path/to/known/hosts> \ <1>
```

Comment:
Maybe "--from-file=<path/to/known_hosts>" it's better, others LGTM

Reply:
@wewang58 Thanks for reviewing. I have made the suggested change. Kindly approve if ok.

Reply:
@snarayan-redhat  Got it.